libqtile/widget/battery.py

```diff
@@ -460,7 +536,11 @@ class Battery(base.ThreadPoolText):
         ("low_foreground", "FF0000", "Font color on low battery"),
         ("low_background", None, "Background color on low battery"),
         ("update_interval", 60, "Seconds between status updates"),
-        ("battery", 0, "Which battery should be monitored (battery number or name)"),
+        (
+            "battery",
```

Comment:
If we're changing the default behaviour, can you add something to the changelog to make this very clear.

Reply:
Sorry for the noob question beacuse its my first contribution to an open source project, i don't use git,... . Do i have to modify the changelog file in the root of the repository?

Reply:
Yes, that's the one!